Meadowlark-js/packages/meadowlark-core/src/api-schema/ApiSchemaLoader.ts

```diff
@@ -12,7 +12,7 @@ import { ProjectNamespace } from '../model/api-schema/ProjectNamespace';
 
 const moduleName = 'ApiSchemaFinder';
 
-const dataStandard50pre1Path: string = path.resolve(__dirname, '../ds-schemas/DataStandard-5.0.0-pre.1.json');
+const dataStandard50Path: string = path.resolve(__dirname, '../ds-schemas/DataStandard-5.0.0.json');
```

Comment:
I wonder if it would make sense to revert to DS 3.3 instead, since we have tests that have "3.3" in the URL, and our descriptors and the sample load process (I assume) are still from 3.3.

Reply:
This is a good point, the bulk load process has:
 $sampleDataVersion = "3.3.1-b"